packages/cypress-commands/src/setups/enableNetworkShim/stubRequests.js

```diff
@@ -47,19 +47,28 @@ function stubRequest(state, request) {
     const requestStub = findMatchingRequestStub(stubProps, state.requestStubs)
 
     if (!requestStub) {
-        const message = `No request stub found for a ${request.method} to "${path}" in test "${testName}". Perhaps you should try a capture run first?`
-        throw new Error(message)
-    }
+        registerMissingRequestStub(state, stubProps)
 
-    const responseBody = getRequesStubResponseBody(requestStub)
+        /*
+         * Reply with an dummy response to avoid the request
+         * from being sent to the destination server
+         */
+        request.reply({
+            body: {},
+            headers: {},
+            statusCode: 200,
```

Comment:
I think 404 is the better status code here, as it's not found.
If there's a manual mock, then this should be irrelevant, right?

Reply:
Great suggestion!